src/cluecode/finder_data.py

```diff
@@ -212,6 +227,11 @@ def classify(s, data_set, suffixes=None):
     if not s:
         return False
     s = s.lower().strip('/')
+    # Separate test for emails - need to ignore xyz@some.com, but not say, xyz@gruesome.com
+    if '@' in s and exact_match:
```

Comment:
Could `ignored_hosts` or similar be a better name for this variable? e.g. using a name that describes what it is, not a name that describes what this used for.

Reply:
Yes, sounds good to me, makes more sense to name it that way.